search-service/src/main/kotlin/com/egm/stellio/search/service/QueryService.kt

```diff
@@ -58,10 +58,10 @@ class QueryService(
         accessRightFilter: () -> String?
     ): Pair<List<CompactedJsonLdEntity>, Int> {
         val temporalEntityAttributes = temporalEntityAttributeService.getForEntities(
-            temporalEntitiesQuery.limit,
-            temporalEntitiesQuery.offset,
-            temporalEntitiesQuery.ids,
-            temporalEntitiesQuery.types,
+            temporalEntitiesQuery.queryParams.limit,
+            temporalEntitiesQuery.queryParams.offset,
+            temporalEntitiesQuery.queryParams.ids ?: emptySet(),
+            temporalEntitiesQuery.queryParams.types ?: emptySet(),
```

Comment:
as it is built, `queryParams` is already an empty set if no `type` parameter is provided.